Library/Recurly.csproj

```diff
@@ -64,6 +64,7 @@
   <ItemGroup>
     <Compile Include="Account.cs" />
     <Compile Include="Address.cs" />
+    <Compile Include="List\UsageList.cs" />

```

Comment:
This is automatically generated, not sure how it works. Would be nice to organize them but it might make a bunch of noise in commits. Will investigate when we get this working without visual studio!